### ISSUE TITLE
Cannot read property 'StopSysSound' of null, js engine: hermes

### ISSUE THREAD

**Issue by GeandersonTodimo** [trietho/react-native-a-beep#28]
package.json
"react": "18.1.0",
"react-native": "0.70.5",
"react-native-a-beep": "^1.2.0",

The following error is occurring.
TypeError: Cannot read property 'StopSysSound' of null, js engine: hermes

**Comment by jackywxd** [comment]
I had the same error and I was able to fix it by moving RNReactNativeABeep.podspec up one level from current folder ios.

**Comment by guptavishesh143** [comment]
> I had the same error and I was able to fix it by moving RNReactNativeABeep.podspec up one level from current folder ios.

@jackywxd  can you please elaborate the steps what you did, Are talking about the IOS level ?

**Comment by jackywxd** [comment]
RNReactNativeABeep.podspec should be moved to folder react-native-a-beep from its subfolder ios

![image](https://user-images.githubusercontent.com/33141943/236577625-e6bbdba3-b2a8-46b4-8d04-036a8effd119.png)

**Comment by QuentG** [comment]
@jackywxd Do you need to apply a patch to do this or something else? I'd love the solution ! 
Thanks ;)

**Comment by Rodrigo816** [comment]
> @jackywxd Do you need to apply a patch to do this or something else? I'd love the solution ! Thanks ;)

Yes you can use https://www.npmjs.com/package/patch-package to patch it.

**Comment by jackywxd** [comment]
If @trietho could fix it in the main branch that's the most ideal solution. For temporary solution, @Rodrigo816 is correct, you could use patch-package to patch it in case you need to run yarn install again. @QuentG

**Comment by simon-assistiq** [comment]
I tried thie `patch-package` + moving podspec approach and it seemed to work fine, but on subsequent pod installs the `Podfile.lock` showed changes removing `RNReactNativeABeep` and the problem would recur... I thought I followed the steps for `patch-package` properly… maybe not?
